code/clue.js

```diff
@@ -41,19 +189,73 @@ const randomSelector = array => {
 // With a killer, a weapon and a room.
 // The values will be set later.
 
+let mystery = {
+  killer:'',
+  weapon:'',
+  room:''
+}
```

Comment:
Any reason for using let instead of const? It's recommended to use const as much as possible if there is not a specific reason for using let.

Reply:
Good point Jessica!
Moreover, super small thing, but I would suggest you to initialise your mystery object with null values rather than empty strings. It's about sending proper information for other developers : null value indicates that probably we will replace these values with something else, but without saying what type will that new value have for now. Initialising your object with empty string values ' ' could be misunderstood by some one in a way that your properties will later on by replaced by new strings only. As a sum up, this would be better :
```
const mystery = {
  killer: null,
  weapon: null,
  room: null,
}
```

Reply:
fixed